ReactWindows/ReactNative.Shared/DevSupport/DevSupportManager.cs

```diff
@@ -533,14 +627,22 @@ public DevOptionHandler(string name, Action onSelect)
 
             public string Name { get; }
 
+#if WINDOWS_UWP
             public IAsyncInfo AsyncInfo { get; set; }
+#else
+            public Action AsyncInfo { get; set; }
```

Comment:
Call it something more accurate than AsyncInfo (like Cancel)?

---
In reply to: [83941933](https://github.com/ReactWindows/react-native-windows/pull/789#discussion_r83941933) [](ancestors = 83941933)